src/nvim/testdir/test_regexp_latin.vim

```diff
@@ -0,0 +1,39 @@
+" Tests for regexp in latin1 encoding
```

Comment:
It was also to simplify merging of future patches, since Bram has been adding different tests to both files.  However, I can drop the latin1 file and just put the unique tests in the utf8 file.